index.css

```diff
@@ -0,0 +1,42 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    border: 2px solid red;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0;
+    height: 100vh;
+}
+
+#app {
+    border: 2px solid green;
+    height: 300px;
+    width: 600px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.loader {
+    border: 16px solid #f3f3f3;
+    /* Light grey */
+    border-top: 16px solid #3498db;
+    /* Blue */
```

Comment:
aaaaa, comment should be always above the thing it related to - I thought you new that already